pkg/controller/secret/secretsync_test.go

```diff
@@ -0,0 +1,186 @@
+package secret
+
+import (
+	"bou.ke/monkey"
+	"fmt"
+	"github.com/IBM/ibm-vpc-block-csi-driver-operator/pkg/util"
+	k8v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"reflect"
+	"testing"
+)
+
+func TestSecretSyncController_translateSecret(t *testing.T) {
```

Comment:
Nit: `TestTranslateSecret`. SecretSyncController is implied by name of the package, no need to repeat it here.